app/src/lib/status-parser.ts

```diff
@@ -23,6 +23,18 @@ export interface IStatusEntry {
   readonly oldPath?: string
 }
 
+export function isStatusHeader(
+  statusItem: StatusItem
+): statusItem is IStatusHeader {
+  return statusItem.kind === 'header'
+}
+
+export function isStatusEntry(
```

Comment:
No need for this function, we can do this inline to reduce the amount of ceremony code.

Reply:
see https://github.com/desktop/desktop/pull/5683#discussion_r222136803